1.html

```diff
@@ -0,0 +1,208 @@
+			name="keywords"
+			content="otc, talks, OTC Talks, Our Tech Community"
+		/>
+		<meta name="description" content="OTC Talks #1" />
+		<meta name="author" content="Harsh Kapadia" />
+		<meta property="og:site_name" content="OTC Talks #1" />
+		<meta property="og:title" content="OTC Talks #1" />
+		<meta property="og:type" content="website" />
+		<meta
+			property="og:description"
+			content="OTC Talks #1 — on any Tech-related topic. Anyone can give a talk and if you're interested, please fill out a form and we'll get in touch with you as soon as possible!"
+		/>
+		<meta property="og:url" content="https://talks.ourtech.community/1" />
+		<meta
+			property="og:image"
+			content="https://talks.ourtech.community/static/img/og-img/home.png"
+		/>
+		<meta property="og:image:width" content="1200" />
+		<meta property="og:image:height" content="600" />
+		<meta property="twitter:card" content="summary_large_image" />
+		<meta
+			property="twitter:image"
+			content="https://talks.ourtech.community/static/img/og-img/home.png"
+		/>
+
+		<script defer src="static/js/script.js"></script>
+
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+			rel="stylesheet"
+		/>
+
+		<link rel="stylesheet" href="static/css/style.css" />
+
+		<link
+			rel="shortcut icon"
+			href="static/img/favicon.ico"
+			type="image/x-icon"
+		/>
+
+		<title>OTC Talks #1</title>
+	</head>
+
+	<body>
+		<header>
+			<nav>
+				<div class="burger-menu">
+					<div class="line-1"></div>
+					<div class="line-2"></div>
+					<div class="line-3"></div>
+				</div>
+
+				<ul class="navbar-ul hide-burger-menu burger-menu-dropdown">
+					<li><a href="/">Home</a></li>
+					<li><a href="/cfp">Talks CFP</a></li>
+					<li><a href="/rules">Rules</a></li>
+					<li>
+						<a
+							href="https://ourtech.community/coc"
+							target="_blank"
+							rel="noreferrer"
+							>Code of Conduct</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://t.me/OurTechComm"
+							target="_blank"
+							rel="noreferrer"
+							>Join us on Telegram</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/two-years"
+							target="_blank"
+							rel="noreferrer"
+							>Two Years of OTC</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://ourtech.community/team"
+							target="_blank"
+							rel="noreferrer"
+							>Our Team</a
+						>
+					</li>
+					<li>
+						<a
+							href="https://links.ourtech.community"
+							target="_blank"
+							rel="noreferrer"
+							>All Links</a
+						>
+					</li>
+				</ul>
+			</nav>
+		</header>
+
+		<h1>OTC Talks #1</h1>
+		<h2>Supercharge your deployments with WASM</h2>
+		<p>
+			by
+			<a
+				href="https://twitter.com/rishit_dagli"
+				target="_blank"
+				rel="noreferrer"
+				>Rishit Dagli</a
+			>
+		</p>
```

Comment:
- How will we handle multiple talks in one OTC Talks session on this page? The page template should look like this is a one talk session and if required, another talk can be easily added. I know this is additional effort right now, but I think it will make automation easier later on if we have an extensible template ready to go.
- I think we should have separate 'Title' and 'Speaker' sections per talk, in case we need to add more information about the speaker or something like that.

Reply:
![image](https://user-images.githubusercontent.com/56108484/212567757-f5bf4f02-e407-47b4-9b1b-f2b62e468010.png)
- Something similar to this, where the summaries can be appended since the list will only get bigger.
- Essentially breaking it into sections where the `OTC Talk number` with a link for that specific talk can be used.
- This sounds similar to how we structure the summaries as well.

Reply:
![image](https://user-images.githubusercontent.com/61280281/212725938-0aff4ae7-707a-4964-b162-74092b17d4e0.png)

What do you think about this?

So, something like this: 

![image](https://user-images.githubusercontent.com/61280281/212726051-136fdcfe-a793-4f8e-8abf-1fd91b44d20c.png)

Reply:
We can have an ordered list as well.

![image](https://user-images.githubusercontent.com/61280281/212726724-7e48bd72-2e55-4972-9464-7e0970801da0.png)

With some padding, we'll be good to go!

Reply:
Yeah the ordered list with padding will work in my opinion.

Reply:
One more thing that I just thought of is that now that we're making provisions for multiple talks in a session, we can have a common 'When and Where' section rather than have that for every talk in a session.

How would that look? Any ideas?

Reply:
Oh yeah, that's true. 
Will inculcate that and get back to you!

Reply:
Please check @HarshKapadia2

Reply:
Nice, but I don't like the center alignment that much.

Current look:
<img src="https://user-images.githubusercontent.com/50140864/214953951-f12791ff-b5e1-42a1-9fc7-e4d75b2958b6.png" width="70%" />

Instead of this, let us have everything left aligned. Let us use `<h2>` for 'When and Where' and for a new section called 'Talk Index', 'Table of Contents' or something like that, which will have a `<ul>` for all the talks.

Does thar make sense to you?

Reply:
Also, can we please move the contact link to the 'When and Where' section and remove it from the 'Abstract'?

Reply:
Another idea is to rename the section 'When and Where' to 'Logistics'.

Reply:
When and Where easier to understand than logistics and we have used it in meetup repo as well

Reply:
Yes, but the 'When and Where' section now has the rules, code of conduct and contact links as well, so renaming 'When and Where' to 'Logistics' makes more sense to me.

Reply:
When and where and Logistics both sound off, went with something more generic.

Continue here: https://github.com/OurTechCommunity/talks/pull/10#issuecomment-1407345136